Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/branch.py

```diff
@@ -169,10 +175,30 @@ def main(cls, args, repository, why=None, redact=False, **kwargs):
             sys.stderr.write("'{}' is an invalid branch name, cannot create it\n".format(args.issue))
             return 1
 
-        remote_re = re.compile('remotes/.+/{}'.format(re.escape(args.issue)))
-        for branch in repository.branches:
-            if branch == args.issue or remote_re.match(branch):
-                sys.stderr.write("'{}' already exists\n".format(args.issue))
+        if args.issue in repository.branches_for(remote=target_remote):
+            if not args.delete_existing:
+                sys.stderr.write("'{}' exists on the remote '{}' and will be overwritten by a push\n".format(args.issue, target_remote))
+                if args.delete_existing is False:
+                    return 1
```

Comment:
--delete-existing's argument string implies that it only affects local branches. Can you change it to clarify what it does to remote branches too?